frontend/src/components/new-car-card/new-car-card.tsx

```diff
@@ -84,7 +90,12 @@ const NewCarCard: React.FC<ExtendedCarCardPropsType> = (props) => {
         >
           <HeartIcon />
         </button>
-        <img className={styles.button} src={compare} alt="compare button" />
+        <img
+          className={styles.button}
+          src={compare}
+          alt="compare button"
+          onClick={handleCompare}
+        />
```

Comment:
`img` is not an interactive component by the HTML semantics. We shouldn't listen events on them